src/components/App/App.js

```diff
@@ -64,7 +65,7 @@ class App extends Component {
             <div className="title-container">
               <h1>FaceIt</h1>
               <Logo />
-              <h2 class="mission-statement">Discover beauty products that compliment our lifestyles.</h2>
+              <h2 class="mission-statement">Discover beauty products that compliment your lifestyle.</h2>
```

Comment:
I think this should be className instead of class. I had an error in my console about this one

Reply:
Oh, you're right! Thank you!